analysis/src/CompletionFrontEnd.ml

```diff
@@ -294,6 +297,7 @@ let rec exprToContextPathInner ~(inJsxContext : bool) (e : Parsetree.expression)
               funct = {pexp_desc = Pexp_ident id; pexp_loc; pexp_attributes};
               args = [(Nolabel, lhs)];
               partial;
+              transformed_jsx;
```

Comment:
perhaps give a name to the inline record passed to `Pexp_apply` in the pattern match, and here only mention the changed cases (so no mention of `partial` and `transformed_jsx`).